resources/migrations/000_migrations.yaml

```diff
@@ -13226,6 +12890,11 @@ databaseChangeLog:
             tableName: collection_permission_graph_revision
             columnName: before
             newDataType: ${text.type}
+      rollback:
+        - modifyDataType:
+            tableName: collection_permission_graph_revision
+            columnName: before
+            newDataType: text
```

Comment:
I don't think we really need to roll these back -- changing from `text` to `longtext` really shouldn't make any difference other than `longtext` having a larger max size

Reply:
Good point.